classes/class-wc-connect-shipping-method.php

```diff
@@ -448,10 +449,11 @@ public function calculate_shipping( $package = array() ) {
 						} else {
 							$this->debug(
 								sprintf(
-									'Received rate: <strong>%s</strong> (%s)<br/><ul><li>%s</li></ul>',
+									'Received rate: <strong>%s</strong> (%s)<br/><ul><li>%s</li></ul><br/>Packing log: <ul><li>%s</li></ul>',
```

Comment:
Can we add i18n here? And also break up this string for readability. Something like this could work:

```
'Received rate: <strong>%s</strong> (%s)<br/>' .
	'<ul><li>%s</li></ul><br/>' .
	'Packing log: ' .
	'<ul><li>%s</li></ul>',
```

Reply:
Reworked in https://github.com/Automattic/woocommerce-services/commit/3dd205cf22692fa4e10cbbfbf9356a756e245bf3.